data/regions.js

```diff
@@ -237,106 +190,261 @@ regions.westernAfrica = {
+    regions.centralAmerica.countries,
+    regions.northernAmerica.countries,
+    regions.caribbean.countries,
+  ])
+  .sort()
+};
+
+regions.southAmerica = {
+  countries: [
+    // source is http://en.wikipedia.org/wiki/South_America
+    'AR', // Argentina
+    'BO', // Bolivia
+    'BR', // Brazil
+    'CL', // Chile
+    'CO', // Colombia
+    'EC', // Ecuador
+    'FK', // Falkland Islands
+    'GF', // French Guiana
+    'GY', // Guyana
+    'PY', // Paraguay
+    'PE', // Peru
+    'SR', // Suriname
+    'UY', // Uruguay
+    'VE', // Venezuela
   ]
 };
 
-regions.pacificIslands = {
+regions.antartica = {
+  countries: [
+    // source is http://en.wikipedia.org/wiki/Antarctica
+    'AQ', // Antarctica
+    'BV', // Bouvet Island
+    'TF', // French Southern Territories
+    'HM', // Heard Island and McDonald Islands
+    'GS', // South Georgia and the South Sandwich Islands
+  ]
+}
+
+regions.northernEurope = {
+  countries: [
+    // source is http://en.wikipedia.org/wiki/Northern_Europe
+    'AX', // Åland
+    'DK', // Denmark
+    'EE', // Estonia
+    'FO', // Faroe Islands
+    'FI', // Finland
+    'GG', // Guernsey
+    'IS', // Iceland
+    'IE', // Republic of Ireland
+    'JE', // Jersey (UK)
+    'IM', // Isle of Man
+    'LV', // Latvia
+    'LT', // Lithuania
+    'NO', // Norway
+    'SJ', // Svalbard and Jan Mayen
+    'SE', // Sweden
+    'GB', // United Kingdom
+  ]
+}
+
+regions.southernEurope = {
   countries: [
-    // source is http://data.worldbank.org/region/PSS
+    // source is http://en.wikipedia.org/wiki/Southern_Europe
+    'AL', // Albania
+    'AD', // Andorra
+    'BA', // Bosnia and Herzegovina
+    'HR', // Croatia
+    'CY', // Cyprus
+    'GI', // Gibraltar
+    'GR', // Greece
+    'IT', // Italy
+    'MK', // Republic of Macedonia
+    'VA', // Vatican City
+    'MT', // Malta
+    'ME', // Montenegro
+    'PT', // Portugal
+    'SM', // San Marino
+    'RS', // Serbia
+    'SI', // Slovenia
+    'ES', // Spain
+  ]
+}
+
+regions.easternEurope = {
+  countries: [
+    // source is http://en.wikipedia.org/wiki/Eastern_Europe
+    'AM', // Armenia
+    'AZ', //Azerbaijan
+    'BY', // Belarus
+    'BG', // Bulgaria
+    'CZ', // Czech Republic
+    'GE', // Georgia
+    'HU', // Hungary
+    'MD', // Moldova
+    'PL', // Poland
+    'RO', // Romania
+    'RU', // Russia
+    'SK', // Slovakia
+    'UA', // Ukraine
+  ]
+}
+
+regions.westernEurope = {
+  countries: [
+    // source is http://en.wikipedia.org/wiki/Western_Europe
+    'AT', // Austria
+    'BE', // Belgium
+    'FR', // France
+    'DE', // Germany
+    'LI', // Liechtenstein
+    'LU', // Luxembourg
+    'MC', // Monaco
+    'NL', // Netherlands
+    'CH', // Switzerland
+  ],
+};
+
+regions.europe = {
+  countries: _.flatten([
+    regions.northernEurope.countries,
+    regions.southernEurope.countries,
+    regions.easternEurope.countries,
+    regions.westernEurope.countries,
+  ])
+  .sort()
+}
+ 
+regions.australia = {
```

Comment:
Should this be `regions.australasia` as suggested by http://en.wikipedia.org/wiki/Australasia? In which case New Guinea should be included somehow...